app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command) == 2:
```

Comment:
It's incorrect. the command is a string that contains the command as `cp file.txt file.text`. You must validate command length after split